apps/builddao/widget/events/Calendar.jsx

```diff
@@ -66,39 +140,76 @@ const handleMonthChange = (change) => {
 const Toolbar = () => {
   return (
     <StyledToolbar>
-      <div className="section gap-3">
-        <span>{dateString}</span>
-        <button className="date-changer" onClick={() => handleMonthChange(-1)}>
-          <i className="bi bi-chevron-left"></i>
-        </button>
-        <button className="date-changer" onClick={() => handleMonthChange(1)}>
-          <i className="bi bi-chevron-right"></i>
-        </button>
-      </div>
-      <div className="section gap-1 justify-content-center">
-        <Button
-          variant={selectedView === "month" ? "secondary" : "outline"}
-          onClick={() => setSelectedView("month")}
-        >
-          Month
-        </Button>
-        <Button
-          variant={selectedView === "list" ? "secondary" : "outline"}
-          onClick={() => setSelectedView("list")}
-        >
-          List
-        </Button>
-      </div>
-      <div className="section justify-content-end" style={{ gap: 10 }}>
-        <Button variant="outline" onClick={() => setShowFilterModal(true)}>
-          Filter By
-        </Button>
-        {context.accountId && (
-          <Button variant="primary" onClick={() => setShowCreateModal(true)}>
-            Add Event
+      <HeaderLeft>
+        <div className="section gap-3">
+          <span>{dateString}</span>
+          <button
+            className="date-changer"
+            onClick={() => handleMonthChange(-1)}
+          >
+            <i className="bi bi-chevron-left"></i>
+          </button>
+          <button className="date-changer" onClick={() => handleMonthChange(1)}>
+            <i className="bi bi-chevron-right"></i>
+          </button>
+          {context.accountId && (
+            <button
+              className="date-changer"
+              style={{
+                border: `1px solid ${theme === "light" ? "#cdcdcd" : "rgba(255, 255, 255, 0.2)"}`,
+                padding: "4px 12px",
+                widht: "32px",
+                height: "32px",
+                borderRadius: "8px",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+              }}
+              onClick={() =>
+                Social.set({
+                  calendar: { theme: theme === "light" ? "dark" : "light" },
+                })
+              }
```

Comment:
Let's replace this with [Storage API](https://docs.near.org/build/near-components/anatomy/web-methods#localstorage)